daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoder.scala

```diff
@@ -130,8 +130,10 @@ private[digitalasset] object DamlLfEncoder extends App {
 
   val MANIFEST =
     """Manifest-Version: 1.0
-      |Created-By: Digital Asset packager
-      |Location: archive.dalf
+      |Created-By: DAML-LF Encoder
+      |Sdk-Version:
```

Comment:
```suggestion
      |Sdk-Version: 0.0.0
```
I’m not quite sure anything will go wrong if you leave this empty but it seems better to use `0.0.0` which corresponds to HEAD.

Reply:
Thanks. I will squeeze this change in a follwoing PR.